examples/tutorial/jupyter/README.md

```diff
@@ -15,8 +15,10 @@ please install every package listed in `requirements.txt` file individually with
 
 ## Run Jupyter Notebooks
```

Comment:
Why was this file moved? We should have a single one for pandas_on_ray and another one for pandas_on_dask in the respective directories. Remember about the OmniSci engine.

Reply:
It was moved to this directory to make it more convenient to user where to start jupyter notebooks - we should start notebooks in the directory where images are located and if we move readme closer to notebooks we will need to tell something like "go in the parent directory, then once again and only then you should start jupyter server". Additionally, if we place readme here we will be capable to avoid bunch of code duplication i believe (for OmniSci specific notes separate section can be created).

Reply:
This file should be corrected with respect to different exexutions.

Reply:
Right, notes for different executions are added.